### DISCUSSION TITLE
How to change location

### DISCUSSION THREAD

**Discussion by anderep** [jasonacox/Powerwall-Dashboard#255] [General]
I found weather411.conf, is that the only file?  What about sunrise/sunset graph?
I didn’t put a negative value I for latitude, so I’m pulling data for China !

**Comment by jasonacox** [comment]
Two areas:

1) Weather - Yes, edit the `weather/weather411.conf` file (or rerun `weather.sh`) and then recreate the docker container:

```bash
docker stop weather411
docker rm weather411
./compose-dash.sh up -d
```

2) Sun and Moon - datasource for graph.  You will need to edit the configuration in Grafana:
Click on settings icon and "Data sources"
 <img width="173" alt="image" src="https://user-images.githubusercontent.com/836718/234178042-2f2b6b91-e17e-48f7-9111-89542957ef7c.png">
 Click on "Sun and Moon"
 <img width="168" alt="image" src="https://user-images.githubusercontent.com/836718/234178130-294d3d10-c093-4f94-b191-c071895e1a57.png">
 Edit the coordinates and save.
